overview/typescript.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TypeScript
+description: TypeScript is an open-source language which builds on JavaScript
+sidebar_position: 8
+---
+
+## Overview
+
+TypeScript is an open-source language which builds on JavaScript, one of the world’s most used tools, by adding static type definitions.
+
+Types provide a way to describe the shape of an object, providing better documentation, and allowing TypeScript to validate that your code is working correctly.
```

Comment:
Are documentation and validation consequences of describing the shape? This is ambiguous and might be better as dot points.

Reply:
Copied it verbatim from TypeScripts own website, do what you want with it.